tests/test_violations/test_implementation.py

```diff
@@ -0,0 +1,19 @@
+import pytest
+
+from dotenv_linter.grammar.parser import DotenvParser
+from dotenv_linter.violations.base import BaseFSTViolation, BaseViolation
+
+
+def test_visitor_returns_location() -> None:
```

Comment:
Not sure, what is the purpose of this. Maybe test method `as_line()`?

Reply:
This file can be safely removed, I think